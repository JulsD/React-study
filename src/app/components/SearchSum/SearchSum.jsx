import React from 'react';

class SearchSum extends React.Component {

  constructor(props) {
    super(props);
    this.state = {
      searchSum: '7 movies found'
    };
  }

  render() {
    return (
      <p>
        {this.state.searchSum}
      </p>
    );
  }

}

export default SearchSum;
