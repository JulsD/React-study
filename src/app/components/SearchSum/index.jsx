import SearchSum from './SearchSum';
export default SearchSum;