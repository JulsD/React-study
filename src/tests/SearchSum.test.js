import React from 'react';
import { SearchSum } from '../app/components/SearchSum/SearchSum';

test('SearchSum should render a <nav>', () => {
  const search = {
    movies: [1,2,3]
  }
  const wrapper = shallow(
    <SearchSum search={search}/>
  );
  expect(wrapper).toMatchSnapshot();
});
