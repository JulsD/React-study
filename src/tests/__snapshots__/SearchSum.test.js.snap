// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchSum should render a <nav> 1`] = `
<p>
  3 movies found
</p>
`;
